ui/app/components/transit-key-actions.js

```diff
@@ -216,7 +216,11 @@ export default Component.extend(TRANSIT_PARAMS, {
       this.toggleProperty('isModalActive');
     },
 
-    doSubmit(data, options = {}) {
+    doSubmit(data, options = {}, maybeEvent) {
+      const event = options.type === 'submit' ? options : maybeEvent;
```

Comment:
This feels... strange. Not a blocker but maybe something we want to come back and clean up (convert to @action perhaps?)

Reply:
Yes I agree. I tried to refactor things as little as possible given the scope. I think it comes down to design when using actions that pass the event to the handler and not having optional args.